R/get_tolid.R

```diff
@@ -280,6 +280,10 @@ tol_fetch_fuzzy <- function(x) {
 }
 
 dtrbsetdf <- function(x) {
+  x <- lapply(x, function(a) {
+    a[sapply(a, is.null)] <- NA
```

Comment:
thanks @gpli 

can you use `NA_character_` instead of `NA` just in case `NA` being a logical may cause some problems

Reply:
@sckott Yes, that does make sense. I missed it.